server/src/api/controllers/project.controller.ts

```diff
@@ -0,0 +1,154 @@
+import { Request, Response } from 'express';
+
+import {
+  getAllProjectItemsService,
+  createOneProjectItemService,
+  getOneProjectItemService,
+  deleteOneProjectItemService
+} from '../services/project.service';
+
+import { success, error } from '../../../node-mongo-helpers';
+
+
+const routeName = 'project';
+const item = `${routeName}-item`;
+
+let response: { [key: string]: unknown } = {};
+
+const getUrl = (req: Request): string => {
+  const protocol = req.protocol;
+  const host = req.hostname;
+  const url = req.originalUrl;
+  const port = process.env.PORT_ATLAS || 3000;
+
+  const fullUrl = `${protocol}://${host}:${port}${url}`
+
+  return fullUrl;
+}
```

Comment:
@samuko-things Nice job with the `getUrl()` method. But help me do this: 
- Create a folder called `helpers` in the root of the `src` folder. Create a file inside this new folder that you created and name the file `methods.ts`. Add your getUrl() method in there, and export and Import the method/file as needed.
- Also, you don't have to create a new port variable. We already have a port somewhere in the server or db file. What you can do is help make the port a global variable that we can access everywhere we need it. In that new helpers folder that you created, create a file in there called variables.js. Declare the port in there and export it everywhere we need it in the application.

Just do this, and I will take over the cleanup from there.

#
> **Note**
> We will be doing this later/soon in the api boilerplate templates repo itself, but not now, since we have 3 templates to update at the same time in there. Once our cleanup is complete here, and we are sure about the approach, we will be implementing it there also. I will let you know when that time comes.